docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -204,9 +204,21 @@ Managed
 ~~~~~~~
 
 You can also run acceptance tests on existing "managed" nodes.
-In this case the configuration file should include:
 
-- **addresses**: The IP addresses of two running nodes.
+This is a quicker way to run the acceptance tests as it avoids the slow process of provisioning new acceptance testing nodes.
+
+The ``managed`` provider:
+
+* re-installs the ``clusterhq-flocker-node`` and ``clusterhq-python-flocker`` packages,
+* creates and distributes new certificates and keys to all the nodes,
+* restarts the ``flocker-control`` on the first ``managed`` node, and
+* restarts the ``flocker-*-agent`` processes on all the nodes.
```

Comment:
Are these details important?  They'll have to be kept up-to-date with the code if they're not to become incorrect details.